### PR TITLE
Legal pages

### PR DESCRIPTION
**Pull request by Hornebom**
Ticket
https://app.zenhub.com/workspaces/centrifuge-applications-5fa29e924059e5001625e2a3/issues/centrifuge/website/546

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -102,5 +102,5 @@
     "skipLibCheck": true /* Skip type checking all .d.ts files. */,
     "resolveJsonModule": true
   },
-  "include": ["./src/**/*", "./gatsby-node.ts", "./gatsby-config.ts", "./plugins/**/*", "**/*.d.ts"]
+  "include": ["./src/**/*", "gatsby-node.js", "./gatsby-config.ts", "./plugins/**/*", "**/*.d.ts"]
```
Comment (onnovisser) [inline tsconfig.json:105]: it's still `gatsby-node.ts` right?

File: src/components/MarkdownSection.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+              {children}
+            </Text>
+          ),
+          p: ({ children }) => (
+            <Text as="p" variant="body1">
+              {children}
+            </Text>
+          ),
+          a: ({ children, href }) => (
+            <Text as="a" href={href} color="textPrimary" style={{ textDecoration: 'underline' }}>
```
Comment (onnovisser) [inline src/components/MarkdownSection.tsx:99]: `rel` missing

File: tsconfig.json (hunk 1 above)
Comment (Hornebom) [reply tsconfig.json]: Yes it is. Removed the reference to `gatsby-node.js`.

File: src/components/MarkdownSection.tsx (hunk 2 above)
Comment (Hornebom) [reply src/components/MarkdownSection.tsx]: Added `rel` attribute and extended this section with internal/external links

File: src/components/MarkdownSection.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                style={{ textDecoration: 'underline' }}
+              >
+                {children}
+              </Text>
+            ) : (
+              <Text
+                as={Link}
+                to={href}
+                color="textPrimary"
+                rel="noopener noreferrer"
```
Comment (onnovisser) [inline src/components/MarkdownSection.tsx:121]: doesn't need the `rel` here

Comment (Hornebom) [reply src/components/MarkdownSection.tsx]: Removed rel attribute on internal Link

**Pull request merged**